internal/filter/chain.go

```diff
@@ -41,37 +41,25 @@ func Chain(filters ...transport.Filter) transport.Filter {
 // filterChain combines a series of filters into a single Filter.
 type chain []transport.Filter
 
-func (c chain) Call(ctx context.Context, request *transport.Request, out transport.Outbound) (*transport.Response, error) {
+func (c chain) Send(ctx context.Context, request *transport.Request, sender transport.RequestSender) (*transport.Response, error) {
```

Comment:
can we name this function something different? I keep forgetting whether the RequestSender interface has a sender passed in.  Maybe just SendChain or something to be very explicitly **not** RequestSender.Send